src/utils/parse.js

```diff
@@ -40,9 +40,11 @@ export function parseIso8601Datetime (datetime) {
   // > Support for ISO 8601 formats differs in that date-only strings
   // > (e.g. "1970-01-01") are treated as UTC, not local.
   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#parameters
-  try {
-    return new Date(datetime)
-  } catch (_e) {
-    return null
```

Comment:
Well would you look at that, my unit test actually found a bug :) Turns out the Date constructor doesn't throw an error if it receives an unparseable date.